chpl/chpl-api/src/main/java/gov/healthit/chpl/questionableActivity/CertificationResultQuestionableActivityProvider.java

```diff
@@ -111,31 +140,39 @@ public QuestionableActivityCertificationResultDTO checkGapUpdated(
         }
         return addedMacras;
     }
-    
-    public List<QuestionableActivityCertificationResultDTO> checkG1MacraMeasuresRemoved(
-            CertificationResult origCertResult, CertificationResult newCertResult) {
-        
-        List<QuestionableActivityCertificationResultDTO> removedMacras = new ArrayList<QuestionableActivityCertificationResultDTO>();
-        if ((newCertResult.getG1MacraMeasures() == null || newCertResult.getG1MacraMeasures().size() == 0) && 
-                origCertResult.getG1MacraMeasures() != null && origCertResult.getG1MacraMeasures().size() > 0) {
+
+    /**
+     * Check for QA re: removal of G1 MACRA measures.
+     * @param origCertResult original certification criteria result
+     * @param newCertResult new certification criteria result
+     * @return list of removed measures
+     */
+     public List<QuestionableActivityCertificationResultDTO> checkG1MacraMeasuresRemoved(
+            final CertificationResult origCertResult, final CertificationResult newCertResult) {
+
+        List<QuestionableActivityCertificationResultDTO> removedMacras
+        = new ArrayList<QuestionableActivityCertificationResultDTO>();
```

Comment:
Should this be indented?  https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent